buster/chatbot.py

```diff
@@ -78,6 +75,12 @@ def __init__(self, cfg: ChatbotConfig):
         self.cfg = cfg
         self._init_documents()
         self._init_unk_embedding()
+        self._init_response_formatter()
+
+    def _init_response_formatter(self):
+        self.response_formatter = ResponseFormatterFactory().get_formatter(
+            format=self.cfg.response_format, response_footnote=self.cfg.response_footnote
+        )
```

Comment:
It would make more sense to either pass the factory object directly or to use a function. Creating an object from a known class, without argument, and then calling a method with the arguments is complicated for no reason.